### DISCUSSION TITLE
Symfony 6.0 and api-platform/api-pack 1.3 conflicts

### DISCUSSION THREAD

**Discussion by gopikrish19** [api-platform/api-platform#2084] [Q&A] [answered]
Hi, 
      I am using using symfony 6.0, and i want to install the api-platform but it throws the conflicts for all of the api platform versions.
 Can api-platform supports with symfony 6.0 ?

=====================================================================================================

--------: composer require api

Using version ^1.3 for api-platform/api-pack
./composer.json has been updated
Running composer update api-platform/api-pack
Loading composer repositories with package information
Restricting packages listed in "symfony/symfony" to "6.0.*"
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - api-platform/core[v1.0.0, ..., v1.1.1] require symfony/serializer ^2.7.2|~3.0 -> found symfony/serializer[v2.7.2, ..., v2.8.52, v3.0.0, ..., v3.4.47] but these were not loaded, likely because it conflicts with another require.
    - api-platform/core[v2.0.0, ..., v2.6.3] require psr/cache ^1.0 -> found psr/cache[1.0.0, 1.0.1] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - api-platform/core[v2.6.4, ..., v2.6.6] require psr/container ^1.0 -> found psr/container[1.0.0, 1.1.0, 1.1.1, 1.1.2] but the package is fixed to 2.0.2 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - api-platform/api-pack v1.3.0 requires api-platform/core * -> satisfiable by api-platform/core[v1.0.0, v1.0.1, v1.1.0, v1.1.1, v2.0.0, ..., v2.6.6].
    - Root composer.json requires api-platform/api-pack ^1.3 -> satisfiable by api-platform/api-pack[v1.3.0].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require api-platform/api-pack:*" to figure out if any version is installable, or "composer require api-platform/api-pack:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

**Comment by dunglas** [comment, marked as answer]
API Platform isn't compatible with Symfony 6 yet but we are working on it: https://github.com/symfony/symfony/pull/44331

**Reply by gopikrish19** [reply]
Thank you

**Reply by gopikrish19** [reply]
Can you please mention the time period, Is it possible to have this bundle with in week?, that will help us to move forward with our project

**Reply by fadel221** [reply]
Oh Thanks
